thirdparty/breakpad/cmake/LinuxBreakpad.cmake

```diff
@@ -0,0 +1,34 @@
+set(SRCS
+        ${SRCS}
+    breakpad/src/client/minidump_file_writer.cc
+    breakpad/src/common/string_conversion.cc
+    breakpad/src/common/convert_UTF.c
+    breakpad/src/common/md5.cc
+
+    breakpad/src/client/linux/log/log.cc 
+    breakpad/src/client/linux/crash_generation/crash_generation_client.cc 
+    breakpad/src/client/linux/dump_writer_common/thread_info.cc 
+    breakpad/src/client/linux/dump_writer_common/ucontext_reader.cc 
+    breakpad/src/client/linux/microdump_writer/microdump_writer.cc 
+    breakpad/src/client/linux/minidump_writer/linux_dumper.cc 
+    breakpad/src/client/linux/minidump_writer/linux_core_dumper.cc 
+    breakpad/src/client/linux/minidump_writer/linux_ptrace_dumper.cc 
+    breakpad/src/client/linux/minidump_writer/minidump_writer.cc 
+    breakpad/src/client/linux/handler/minidump_descriptor.cc 
+    breakpad/src/client/linux/handler/exception_handler.cc 
+    breakpad/src/common/linux/guid_creator.cc 
+    breakpad/src/common/linux/file_id.cc 
+    breakpad/src/common/linux/elfutils.cc 
+    breakpad/src/common/linux/memory_mapped_file.cc 
+    breakpad/src/common/linux/safe_readlink.cc 
+    breakpad/src/common/linux/linux_libc_support.cc
+    breakpad/src/common/linux/elf_core_dump.cc
+    #breakpad/src/common/linux/elf_core_dump.cc
+
+	#crash sender
+        #breakpad/src/common/linux/google_crashdump_uploader.cc
+)
+
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS 1)
```

Comment:
Only do this on Windows. Also what is the reasoning for wanting this as a shared lib?